docs/discover/search.asciidoc

```diff
@@ -88,6 +88,81 @@ If the saved search is associated with a different index pattern than is current
+
+[role="xpack"]
+==== Read only access
+If you have insufficient privileges to save queries, the *Save current query* button isn't visible in the saved query management popover.
+
+[role="screenshot"]
+image::discover/images/saved-query-management-component-readonly-view.png["Example of the saved query management popover with read only access to Discover",width="80%"]
+
+==== Saving a query
+To save the current query text, filters, and time filter:
+
+. Click *#* in the search bar, next to the query text input.
+. Click *Save current query* in the popover.
++
+[role="screenshot"]
+image::discover/images/saved-query-management-component-all-privileges.png["Example of the saved query management popover with a list of saved queries with write access",width="80%"]
++
+. Enter a name, a description, and then select the filter options that you want to include. By default filters are automatically included, but the time filter is not.
++
+[role="screenshot"]
+image::discover/images/saved-query-save-form-default-filters.png["Example of the saved query management save form with the filters option included and the time filter option excluded",width="80%"]
+. Click *Save*.
+
+==== Loading a query
+To load a saved query into Discover, Dashboard, or Visualize:
+
+. Click *#* in the search bar, next to the query text input.
+. Select the query you want to load. You might need to scroll down to find the query you are looking for.
+
+==== Saving changes to a query
+If you load a query and then make changes to the query text, the filters, or the time filter, you can save the changes as a new query or update the existing query.
+
+To save the changes as a new query:
+
+. Click *#* in the search bar, next to the query text input.
+. Click *Save as new* in the popover.
+. Enter a name and a description, and then select the filter options that you want to include.
+. Click *Save*.
++
+[role="screenshot"]
+image::discover/images/saved-query-management-component-save-as-new-query.png["Example of the saved query management popover when a query is loaded and we have made changes to the query",width="80%"]
+
+To save the changes to the current query:
+
+. Click *#* in the search bar.
+. Click *Save changes* in the popover.
+. Enter a description, and then select the filter options that you want to include.
+. Click *Save*.
+
+==== Clearing a query
+To clear a query that is currently loaded in an application:
+
+. Click *#* in the search bar.
+. Click *Clear* in the popover.
+
+==== Deleting a query
+To completely delete a query:
+
+. Click *#* in the search bar, next to the query text input.
+. Hover on the query you want to delete.
+. Click the trash can icon to delete the query.
++
+[role="screenshot"]
+image::discover/images/saved-query-management-component-delete-query-button.png["Example of the saved query management popover when a query is hovered over and we are about to delete a query",width="80%"]
```

Comment:
I don't feel this image is necessary

Reply:
I feel it is needed because the trash can icon doesn't appear in any of the other images. It's a subtle difference in the visual state in the component.